### ISSUE TITLE
double.TryParse returns wrong result when input contains + or -

### ISSUE THREAD

**Issue by aks427** [bridgedotnet/Bridge#3979]
`double.TryParse()` is returning a different result in Bridge than .Net when it attempts to parse a string that contains a + or - character.

This issue is very similar to a previously fixed issue: https://github.com/bridgedotnet/Bridge/issues/3360
Which was resolved with this PR: https://github.com/bridgedotnet/Bridge/pull/3413

It looks like all cases were fixed except + and -.

### Steps To Reproduce

https://dev.deck.net/d00bfd1642aeda7fb3b8b65d7654dfe0
https://dotnetfiddle.net/DALfVi

```csharp
public class Program
{
    public static void Main()
    {
	double test;
      	if (double.TryParse("2+1", out test))
        {
            Console.WriteLine(test);
        }
        else
        {
            Console.WriteLine("Invalid");
        }
    }
}

```

### Expected Result

```js
Invalid
```

### Actual Result

```js
2
```

**Comment by geoffreymcgill** [comment]
@aks427 – Thanks for the report. I've tagged as a defect and we will investigate right away.

**Comment by geoffreymcgill** [comment]
A fix for this issue has been created. After a final review and testing the fix will be included in the upcoming Bridge 17.8.0 release. Thanks again for reporting the defect.

**Comment by fabriciomurta** [comment]
The fix looks good and was merged upstream.
